src/ConnectedVMware/README.md

```diff
@@ -62,7 +62,8 @@ directive:
     remove: true
   # Remove the set-* cmdlet
   - where:
-      verb: Set
```

Comment:
Why you want to keep set-* cmdlets?

Reply:
seems this auto generated, I have not done this.

Reply:
I did this, all Set-* cmdlets should be removed by default